h2o-core/src/main/java/water/fvec/Vec.java

```diff
@@ -460,6 +460,15 @@ public static Vec makeCon(double x, long len, int log_rows_per_chunk, boolean re
 //    System.out.println("made vecs " + Arrays.toString(res));
     return res;
   }
+  
+  public Vec makeOneDoubles(double value) {
```

Comment:
Is this method used? I cannot fined where it's called.

Reply:
No, it is not used and I deleted it.